html/semantics/embedded-content/the-img-element/relevant-mutations.html

```diff
@@ -165,46 +176,92 @@
     img.src = '/images/green-2x2.png';
   }, 'load');
 
-  t('crossorigin absent to empty', function(img) {
+  // Without an src, changing the crossorigin attribute state SHOULD NOT
```

Comment:
a src* (and MUST NOT, if you wanna do RFC2119)

Reply:
Oops! I originally wrote this "an image", and then swapped it to "src".